### ISSUE TITLE
Alternatives to the firebase.getValue () command

### ISSUE THREAD

**Issue by staincodes** [EddyVerbruggen/nativescript-plugin-firebase#1253]
On a vue web application you can run this firebase callback.
In your nativescript plugin you can get a similar callback, which allows me to get the data every time you change it?
Because currently with the getValue () function I can do the request only once.
```
data.child("name").on("value", function(snapshot){
        this.example = snapshot.val();
      })
```

**Comment by EddyVerbruggen** [comment]
Firestore: https://github.com/EddyVerbruggen/footplr/blob/e43e347c4da6cb5ac39f35bd9d23deb55b98ef6b/app/services/EditingUserService.ts#L26-L33

Realtime DB: https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/c6aef9edd448f58c67463c15915c5b0c1623904e/docs/DATABASE.md#addchildeventlistener

**Comment by staincodes** [comment]
Thank you for your reply,
I'm using nativescript-vue and my intention was to insert the callback / function on mouted ().
Recently used nativescript is I have several difficulties...
Can it work like this? My goal is to update in real time a data that I visualize in the app.
```
mounted(){
var onChildEvent = function(result) {
        //console.log("Event type: " + result.type);
        //console.log("Key: " + result.key);
        //console.log("Value: " + JSON.stringify(result.value));
        this.dataRef = JSON.stringify(result.value);
      };

      firebase.addChildEventListener(onChildEvent, "/data").then(
        function(listenerWrapper) {
          var path = listenerWrapper.path;
          var listeners = listenerWrapper.listeners; /
        }
      );
}
```

**Comment by EddyVerbruggen** [comment]
Yeah that should probably work, but it's beyond the scope of this plugin. Are those commented console logs being invoked?

**Comment by staincodes** [comment]
I removed the comment from the console.log (), I tried to execute the code. But unfortunately I don't get anything out. I have a problem with the user field with some data in it, but nothing works.

**Comment by EddyVerbruggen** [comment]
Does the demo app in this repo work for you? Not the vue / angular one, the ome in the demo folder.

**Comment by staincodes** [comment]
If you mean [this](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/tree/master/demo-vue, ) yes...

**Comment by staincodes** [comment]
The problem is that I have to create an app that reads the temperature in real time of a place. Usually when it comes to a web app with vue it is very easy to make a callback with snapshot.val () ... I would like to get the same result on nativescript-vue.

**Comment by EddyVerbruggen** [comment]
I meant actually running the app in the `demo` folder, and executing [this code](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/c6aef9edd448f58c67463c15915c5b0c1623904e/demo/app/main-view-model.ts#L1364). I guess that's what you're trying to do in Vue. I was wondering if that code works for you, and if it does I don't see why that would not work in Vue as well.

**Comment by staincodes** [comment]
The console gives me this error...
```
'firebase.addChildEventListener error: Run init() first!'
```

**Comment by EddyVerbruggen** [comment]
That the button at the top of the app.
